src/fhost_fpga.py

```diff
@@ -336,6 +336,23 @@ def qdr_okay(self):
         LOGGER.info('%s: QDR okay.' % self.host)
         return True
 
+    def check_fft_overflow(self):
+        """
+        Checks if pfb counters on f-eng are not incrementing
+        :return: True/False
+        """
+        for cnt in range(0, 1):
+            overflow0 = self.registers.pfb_ctrs.read()['data']['pfb_of%d_cnt' % cnt]
+            time.sleep(1)
```

Comment:
Don't think we need to sleep this long? We should be processing 1000's of spectra per second, so sleeping a couple of milliseconds should be fine?